src/framework/application.js

```diff
@@ -2015,14 +2015,37 @@ Object.assign(Application.prototype, {
     /**
      * @function
      * @name pc.Application#renderLines
-     * @description Draw an array of lines.
-     * @param {pc.Vec3[]} position - An array of points to draw lines between.
-     * @param {pc.Color[]} color - An array of colors to color the lines. This must be the same size as the position array.
+     * @description Renders an arbitrary number of discrete line segments. The lines
+     * are not connected by each subsequent point in the array. Instead, they are
+     * individual segments specified by two points. Therefore, the lengths of the
+     * supplied position and color arrays must be the same and also must be a multiple
+     * of 2. The colors of the ends of each line segment will be interpolated along
+     * the length of each line.
```

Comment:
I be tempted to say that each pair of points represents a single line rather than using the word segments. Although, the example does make it clearer below.

Reply:
Yeah, I used segments to indicate that the resulting line isn't necessarily continuous. It was actually quite tricky to draw up a clear, concise description, but let's go with this for now. It's better than what was there before. 😄